indexer/main.go

```diff
@@ -100,8 +101,15 @@ func main() {
 	log.Println("Started retrieving versions for go packages\n")
 
 	var goErrorPackageMap = make(map[string]*GoPackage)
+	cluster := gocql.NewCluster("gophr.dev")
+	cluster.ProtoVersion = 4
+	cluster.Keyspace = "excelsior"
+	cluster.Consistency = gocql.One
+	session, _ := cluster.CreateSession()
+	defer session.Close()
```

Comment:
Also on that note is it maybe better to create one insert query that gets generated via the loop through all the packages. (As opposed to 28,000 individual inserts)